### PR TITLE
Fix recommended protoc version

### PR DESCRIPTION
**Pull request by tpaschalis**
A tiny PR to synchronize the `protoc` version the prompb docs suggests using, with the one checked against in the genproto.sh script.

### CODE REVIEW

File: prompb/README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ re-compile them when building Prometheus.
 If however you have modified the defs and do need to re-compile, run
 `make proto` from the parent dir.
 
-In order for the script to run, you'll need `protoc` (version 3.12.3) in your
-PATH.
+In order for the [script][] to run, you'll need `protoc` (version 3.15.8) in
```
Comment (roidelapluie) [inline prompb/README.md:7]:
```suggestion
In order for the [script][../scripts/genproto.sh] to run, you'll need `protoc` (version 3.15.8) in
```

I don't think we need to make this more complex

File: prompb/README.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ re-compile them when building Prometheus.
 If however you have modified the defs and do need to re-compile, run
 `make proto` from the parent dir.
 
-In order for the script to run, you'll need `protoc` (version 3.12.3) in your
-PATH.
+In order for the [script][../scripts/genproto.sh] to run, you'll need `protoc` (version 3.15.8) in
+your PATH.
+
+[script]: ../scripts/genproto.sh
```
Comment (roidelapluie) [inline prompb/README.md:10]: now that line is extra

Comment (tpaschalis) [reply prompb/README.md]: 🤦 apologies, done

File: prompb/README.md [hunk 3]
```diff
@@ -4,6 +4,6 @@ re-compile them when building Prometheus.
 If however you have modified the defs and do need to re-compile, run
 `make proto` from the parent dir.
 
-In order for the script to run, you'll need `protoc` (version 3.12.3) in your
-PATH.
+In order for the [script][../scripts/genproto.sh] to run, you'll need `protoc` (version 3.15.8) in
```
Comment (roidelapluie) [inline prompb/README.md:7]:
```suggestion
In order for the [script](../scripts/genproto.sh]) to run, you'll need `protoc` (version 3.15.8) in
```

Sorry!

Comment (tpaschalis) [reply prompb/README.md:7]: I think this works in Github-flavored markdown as well? But yeah. Sorry for spending so much time in such a silly PR :P

**Comment by roidelapluie** [comment]
Thanks!

**Pull request merged**